pygmo/docstrings.cpp

```diff
@@ -1087,7 +1087,7 @@ The best known solution for the )"
 
 std::string translate_docstring()
 {
-    return R"(__init__(udp = null_problem(), translation = [0.])
+    return R"(__init__(udp, translation)
```

Comment:
Yes, that's why I added the blurb at the bottom about permitting certain sets of arguments for the constructor (2 or 3 + 1 optional). It seemed the easier way to document the actual behaviour of the class without going around re-implementing constructors on the Python side (unfortunately I don't have any better idea that does not involve considerable effort to emulate the behaviour of C++ functions in Python with respect to overloading).

At least with the def argument in the docstring we are documenting which problem is initialised in case the def ctor is called.